artnet/__init__.py

```diff
@@ -89,9 +89,21 @@ def get_items(self):
         """
         Returns added items
 
-        :return: array of items held by the device
+        :return: array of items held by the device, sorted by their DMX-address
+        """
+        instance_key = "artnet_address"
+        if self._instance_name:
+            instance_key = "artnet_address@"+self._instance_name
+
+        return sorted(self._items, key=lambda i: i.conf[instance_key])
```

Comment:
To access the item's configuration you should use the `plugin.get_iattr_value(i.conf, 'artnet_address')`. This will take care about the instance name. When changing the logic how we handle instance names in configuration your solution will break.

Reply:
Yes, after now having gained more experience with writing PlugIns I will do so...

Reply:
solved it